client/my-sites/checkout/composite-checkout/components/jetpack-checkout-sidebar-plan-upsell/index.tsx

```diff
@@ -0,0 +1,205 @@
+	return (
+		<>
+			<div
+				className={ classNames(
+					'checkout-sidebar-plan-upsell__plan-grid-cell',
+					isTitle ? 'section-title' : ''
+				) }
+			>
+				{ labelMarkup }
+			</div>
+			<div
+				className={ classNames(
+					'checkout-sidebar-plan-upsell__plan-grid-cell',
+					isTitle ? 'section-title' : ''
+				) }
+			>
+				{ valueMarkup }
+			</div>
+		</>
+	);
+};
+
+const JetpackCheckoutSidebarPlanUpsell: FC = () => {
+	const { formStatus } = useFormStatus();
+	const isFormLoading = FormStatus.READY !== formStatus;
+	const reduxDispatch = useDispatch();
+	const { __ } = useI18n();
+	const cartKey = useCartKey();
+	const { responseCart, replaceProductInCart } = useShoppingCart( cartKey );
+	const plan = responseCart.products.find(
+		( product ) => isJetpackProduct( product ) || isJetpackPlan( product )
+	);
+
+	const variants = useGetProductVariants( plan );
+
+	if ( ! plan ) {
+		debug( 'no jetpack plan found in cart' );
+		return null;
+	}
+
+	const currentVariant = variants?.find( ( product ) => product.productId === plan.product_id );
+	const biennialVariant = variants?.find( ( product ) => product.termIntervalInMonths === 24 );
+
+	if ( ! biennialVariant ) {
+		debug( 'plan in cart has no biennial variant; variants are', variants );
+		return null;
+	}
+
+	if ( ! currentVariant ) {
+		debug( 'plan in cart has no current variant; variants are', variants );
+		return null;
+	}
+
+	if ( biennialVariant.productId === plan.product_id ) {
+		debug( 'plan in cart is already biennial' );
+		return null;
+	}
+
+	const onUpgradeClick = () => {
+		if ( isFormLoading ) {
+			return;
+		}
+		const newPlan = {
+			product_slug: biennialVariant.productSlug,
+			product_id: biennialVariant.productId,
+		};
+		debug( 'switching from', plan.product_slug, 'to', newPlan.product_slug );
+		reduxDispatch(
+			recordTracksEvent( 'calypso_jetpack_checkout_sidebar_upsell_click', {
+				upsell_type: 'biennial-plan',
+				switching_from: plan.product_slug,
+				switching_to: newPlan.product_slug,
+			} )
+		);
+		replaceProductInCart( plan.uuid, newPlan );
+	};
+
+	const percentSavings = getItemVariantDiscountPercentage( biennialVariant, currentVariant );
+	if ( percentSavings === 0 ) {
+		debug( 'percent savings is too low', percentSavings );
+		return null;
+	}
+
+	const isComparisonWithIntroOffer =
+		biennialVariant.introductoryInterval === 2 &&
+		biennialVariant.introductoryTerm === 'year' &&
+		currentVariant.introductoryInterval === 1 &&
+		currentVariant.introductoryTerm === 'year';
+	const currencyConfig = {
+		stripZeros: true,
+		isSmallestUnit: true,
+	};
+
+	const cardTitle = createInterpolateElement(
+		sprintf(
+			// translators: "percentSavings" is the savings percentage for the upgrade as a number, like '20' for '20%'.
+			__( '<strong>Save %(percentSavings)d%%</strong> by paying for two years' ),
+			{ percentSavings }
+		),
+		{ strong: createElement( 'strong' ) }
```

Comment:
Minor, but I _think_ you can use JSX for this.
```suggestion
		{ strong: <strong /> }
```

Reply:
Nice catch! This was copied from the initial 2 year plans breakdown component and I did not catch that